.env

```diff
@@ -22,7 +22,7 @@ INDEXER_SHA=""
 
 # Sandbox configuration:
 SANDBOX_URL="https://github.com/algorand/sandbox"
-SANDBOX_BRANCH="master"
+SANDBOX_BRANCH="experimental-api"
```

Comment:
TODO: set this back to master (Theoretically, we won't need to support experimental flags on sandbox when the related simulate endpoint PR is merged into master)